core/src/main/java/tech/tablesaw/columns/numbers/DoubleParser.java

```diff
@@ -24,9 +26,15 @@ public boolean canParse(String s) {
       return true;
     }
     try {
-      Double.parseDouble(AbstractColumnParser.remove(s, ','));
+      if (isPercent(AbstractColumnParser.remove(s, ','))) {
+        s = AbstractColumnParser.remove(s, ',');
+        NumberFormat numberFormat = NumberFormat.getPercentInstance();
```

Comment:
I think you could combine this line and the next into a single line:
```
NumberFormat.getPercentInstance().parse(s);
```

As it is now, `double d` looks to be unused

Reply:
Yes, it simplify code. I have modified it. Thx for suggestion.